api/query/search.go

```diff
@@ -211,6 +216,11 @@ func (sh structuredSearchHandler) ServeHTTP(w http.ResponseWriter, r *http.Reque
 	}
 	runIDsStr := strings.Join(runIDStrs, ",")
 	r2.URL.RawQuery = fmt.Sprintf("run_ids=%s&q=%s", url.QueryEscape(runIDsStr), url.QueryEscape(simpleQ.Pattern))
+
+	if _, showMetadata := q[shared.MetadataKey]; showMetadata {
```

Comment:
use the utility for this (I think it's shared.ParseBooleanParam or similar). ?foo=false would still be "present", but should not be considered true.